src/codegen/llvm/codegen_llvm_visitor.cpp

```diff
@@ -11,6 +11,8 @@
 #include "visitors/rename_visitor.hpp"
 #include "visitors/visitor_utils.hpp"
 
+#include "llvm/Analysis/TargetLibraryInfo.h"
+#include "llvm/CodeGen/ReplaceWithVeclib.h"
```

Comment:
@pramodk Maybe alternatively we can have `#ifndef LLVM 13 ...` -> replace with vector library, else just have a warning that vector maths functions cannot be replaced. What do you think?

Reply:
I was thinking to suggest that but thought if it would become complicated.

But given that we are using trunk, if someone want to quickly try without veclib replace part then i think it’s useful to have this flexibility to build with older version. So let’s do that!

Reply:
@pramodk Done!